EIPS/eip-9174.md

```diff
@@ -0,0 +1,144 @@
+---
+eip: 9174
```

Comment:
```suggestion
eip: 7848
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates and are no longer the issue number.

Please also update the filename.